keps/sig-storage/1472/README.md

```diff
@@ -0,0 +1,1416 @@
+- [Production Readiness Review Questionnaire](#production-readiness-review-questionnaire)
+  - [Feature enablement and rollback](#feature-enablement-and-rollback)
+  - [Rollout, Upgrade and Rollback Planning](#rollout-upgrade-and-rollback-planning)
+  - [Monitoring requirements](#monitoring-requirements)
+  - [Dependencies](#dependencies)
+  - [Scalability](#scalability)
+  - [Troubleshooting](#troubleshooting)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+  - [CSI drivers without topology support](#csi-drivers-without-topology-support)
+  - [Storage class parameters that never affect capacity](#storage-class-parameters-that-never-affect-capacity)
+  - [Multiple capacity values](#multiple-capacity-values)
+  - [Node list](#node-list)
+  - [CSIDriver.Status](#csidriverstatus)
+    - [Example: local storage](#example-local-storage-1)
+    - [Example: affect of storage classes](#example-affect-of-storage-classes-1)
+    - [Example: network attached storage](#example-network-attached-storage-1)
+  - [CSIStoragePool](#csistoragepool)
+      - [Example: local storage](#example-local-storage-2)
+      - [Example: affect of storage classes](#example-affect-of-storage-classes-2)
+      - [Example: network attached storage](#example-network-attached-storage-2)
+  - [Prior work](#prior-work)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+<!--
+**ACTION REQUIRED:** In order to merge code into a release, there must be an
+issue in [kubernetes/enhancements] referencing this KEP and targeting a release
+milestone **before the [Enhancement Freeze](https://git.k8s.io/sig-release/releases)
+of the targeted release**.
+
+For enhancements that make changes to code or processes/procedures in core
+Kubernetes i.e., [kubernetes/kubernetes], we require the following Release
+Signoff checklist to be completed.
+
+Check these off as they are completed for the Release Team to track. These
+checklist items _must_ be updated for the enhancement to be released.
+-->
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [ ] (R) Graduation criteria is in place
+- [ ] (R) Production readiness review completed
+- [ ] Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
```

Comment:
Can you check the ones already done?

Reply:
I'm not sure which ones I can consider done. I've marked those where I am relatively sure.